src/main/java/org/kainos/ea/api/JobService.java

```diff
@@ -0,0 +1,23 @@
+package org.kainos.ea.api;
+
+import org.kainos.ea.cli.Job;
+import org.kainos.ea.client.CantGetAnyRolesException;
+import org.kainos.ea.db.DatabaseConnector;
+import org.kainos.ea.db.JobDao;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class JobService {
+
+  private JobDao jobDao = new JobDao();
```

Comment:
You don't need to initialise the `JobDao` here, it should be passed in as part of the constructor